src/wasm/asm-wasm-builder.cc

```diff
@@ -524,6 +527,7 @@ class AsmWasmBuilderImpl : public AstVisitor {
   Isolate* isolate_;
   Zone* zone_;
   TypeCache const& cache_;
+  int depthInFunction_;
```

Comment:
@aseemgarg89 for naming conventions, see: https://www.chromium.org/developers/coding-style#Naming and https://google.github.io/styleguide/cppguide.html#Naming. Basically, snake_case for variable names, CamelCase for function names and class names.